### ISSUE TITLE
"Include unexpected" error with string include

### ISSUE THREAD

**Issue by seally1186** [sequelize/sequelize#10613]
## What are you doing?
Using a string include in, e.g. `findAll`

```js
  const sequelize = new Sequelize(config.dbUrl);
  const a = sequelize.define("a", { x: DataTypes.INTEGER });
  const b = sequelize.define("b", { y: DataTypes.INTEGER });
  b.belongsTo(a);
  await sequelize.sync({ force: true });
  await b.findAll({ include: ["a"] });
```

## What is actually happening?
```
Error: Include unexpected. Element has to be either a Model, an Association or an object.
    at Function._conformInclude (/var/project/node_modules/sequelize/lib/model.js:390:11)
    at options.include.options.include.map.include (/var/project/node_modules/sequelize/lib/model.js:326:59)
    at Array.map (<anonymous>)
    at Function._conformOptions (/var/project/node_modules/sequelize/lib/model.js:326:39)
    at Function._baseMerge (/var/project/node_modules/sequelize/lib/model.js:792:10)
    at Function._defaultsOptions (/var/project/node_modules/sequelize/lib/model.js:823:31)
    at Function._injectScope (/var/project/node_modules/sequelize/lib/model.js:3104:10)
    at Promise.try (/var/project/node_modules/sequelize/lib/model.js:1697:12)
    at tryCatcher (/var/project/node_modules/bluebird/js/release/util.js:16:23)
    at Function.Promise.attempt.Promise.try (/var/project/node_modules/bluebird/js/release/method.js:39:29)
    at Function.findAll (/var/project/node_modules/sequelize/lib/model.js:1696:23)
    at /var/project/test.js:16:11
```

It seems like when `_baseMerge` calls `_conform_options`, it doesn't pass (and doesn't have) the model, so `_transformStringAssociation` can't change the include.

(Seems like commenting out line 792 of model.js works, as `_confirmOptions` is called later by `findAll`, but not sure if there are other implications.)

__Dialect:__ any (tested on mysql and mariadb)
__Dialect version:__ any
__Database version:__ any
__Sequelize version:__ `5.1.1` and `master`
__Tested with latest release:__ Yes

**Comment by SimonSchick** [comment]
You need to pass `{ as: 'a' }` to `belongsTo`.

**Comment by seally1186** [comment]
Like this?
```
  const sequelize = new Sequelize(config.dbUrl);
  const a = sequelize.define("a", { x: DataTypes.INTEGER });
  const b = sequelize.define("b", { y: DataTypes.INTEGER });
  b.belongsTo(a, { as: "a" });
  await sequelize.sync({ force: true });
  await b.findAll({ include: ["a"] });
```
Getting the same error.

It seems like `{ as: "a" }` is the default?
```
> b.associations.a.as
'a'
```
(with the original code)

**Comment by seally1186** [comment]
This is fixed in 5.2.0.
